### PR TITLE
Doc for the Point Output Process

### PR DESCRIPTION
**Pull request by roigcarlo**
**📝 Description**
Adds the doc for the `Point Output Process`

@juancamarotti Doc for yesterday's process :) 
@jcotela I added you becase vscode tells me that the process was added by you, but feel free to ignore if its not the case.

### CODE REVIEW

File: docs/pages/Kratos/Processes/Output_Process/Point_output_process.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+This process writes results from a geometrical position (point) in the model to a file. It first searches the entity containing the requested output location and then interpolates the requested variable(s). The output can be requested for elements, conditions and nodes. For nodes no geometrical interpolation is performed, the exact coordinates have to be specified. This process works in MPI as well as with restarts. It can serve as a basis for other processes (e.g. MultiplePointsOutputProcess). Furthermore it can be used for testing in MPI where the node numbers can change
+
+## Execution
+
+This process is executed in the following hooks:
+
+##### `ExecuteInitialize`
+
+Initializes the search structures, the interal search strategy (Normal, Restarted, MPI).
```
Comment (juancamarotti) [inline docs/pages/Kratos/Processes/Output_Process/Point_output_process.md]: internal*

File: docs/pages/Kratos/Processes/Output_Process/Point_output_process.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+##### `ExecuteInitializeSolutionStep`
+
+Search for the entitites to be printed. This operation is done for every time step loop in case the entitites have changed position.
```
Comment (juancamarotti) [inline docs/pages/Kratos/Processes/Output_Process/Point_output_process.md]: entities*

**Review by juancamarotti — DISMISSED** [review]

**Comment by jcotela** [comment]
@roigcarlo I no longer remember if I was the original author of this class, but the documentation looks good to me. Probably a good idea to fix the typos that @juancamarotti indicates, I do not have much more to add other than that.

**Review by jcotela — APPROVED** [review]
Re-approving

**Review by juancamarotti — APPROVED** [review]

**Review by juancamarotti — APPROVED** [review]

**Review by juancamarotti — APPROVED** [review]
Everything looks fine :)

**Pull request merged**